composer.json

```diff
@@ -24,7 +24,7 @@
     },
     "require": {
         "php": ">=5.5",
-        "symfony/symfony": "^2.7",
+        "symfony/symfony": "2.7.*",
```

Comment:
Just try phpunit with SF 2.8! :smiley: 

It seems some BC breaks are here finally. :-/